tests/cases/fourslash/jsDocContextualTagsInJsDoc1.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference path="fourslash.ts" />
+
+//// /**
+////  * @param x/*1*/ Does the thing
+////  */
+//// function foo(x/*2*/) {}
+
+verify.quickInfoAt("1", "(parameter) x: any", "Does the thing", undefined);
```

Comment:
This implements the change described by @mjbvz [here](https://github.com/microsoft/TypeScript/issues/59402#issuecomment-2246456638). Maybe *some* JSDoc tags should be returned when requesting quick info for a JSDoc node? I don't know when it would be desirable though. It seems that they always describe an associated node and not itself. The "itself" description gets still returned as part of the documentation string